nixd/include/nixd/AST/ParseAST.h

```diff
@@ -50,9 +50,18 @@ class ParseAST {
 
   void prepareDefRef();
 
+  /// Maps nix expression pointers back, to the name of attribute
+  ///
+  /// { foo  = "bar"; }
+  ///   ^~~ <- ^~~~
+  std::map<const nix::Expr *, nix::Symbol> AttrNamesMap;
+
+  void getAttrNamesMap();
```

Comment:
```suggestion
  void constructAttrNamesMap();
```

rename suggestion because "getXXX" implies it returns some value